### PR TITLE
DINT-28: add AEP + EventForwarding schemas to SDK

### PR DESCRIPTION
**Pull request by aniham**

### CODE REVIEW

**Comment by bradgarropy** [comment]
👀👀👀

File: src/types/schemas/eventForwarding.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+export type EventForwarding = {
+    snowplow?: boolean;
+    aep?: boolean;
+};
```
Comment (bradgarropy) [inline src/types/schemas/eventForwarding.ts:9]: Hi, hope everyone is doing great! One question here, are they are default values for this context? I'm assuming we wouldn't want to start with `aep` and `snowplow` as `false`.

Comment (aniham) [reply src/types/schemas/eventForwarding.ts:9]: lol!! @bradgarropy nice to have you here! t

he recommended values and the way the collector will default is snowplow true (so nothing gets disrupted from what we already do), aep false as it's an optional service a merchant has to set up + pay for :)

Comment (aniham) [reply src/types/schemas/eventForwarding.ts:9]: i'll tag you on the collector pr once it's ready for full review - it's dependent on this change obvi

if forwarding for aep is `true` but the ims org + datastream aren't set (or are set to values that the adobe web sdk collector doesn't like), the adobe collector ([alloy](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html?lang=en)) just won't get initialized and won't send

**Review by benjaminkalk — APPROVED** [review]
Still not super familiar with TypeScript, but everything looks pretty OK and straightforward to me.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,14 @@ The SDK API is broken down into four major parts: [Context][context], [Publish][
 
 These functions get and set context data.
 
+#### `mse.context.getAEP`
+
+Gets the `AEP` (Adobe Experience Platform) context.
```
Comment (evargast) [inline README.md:69]: Hola! 

This is the first time AEP is referenced, should we maybe link the documentation of AEP in the readme? 
```suggestion
Gets the `AEP` ([Adobe Experience Platform](https://business.adobe.com/products/experience-platform/adobe-experience-platform.html)) context.
```

File: src/types/schemas/aep.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
```
Comment (evargast) [inline src/types/schemas/aep.ts:2]: 👀 
```suggestion
 * Copyright © Adobe. All rights reserved.
```

File: src/types/schemas/eventForwarding.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
```
Comment (evargast) [inline src/types/schemas/eventForwarding.ts:2]: x2

```suggestion
 * Copyright © Adobe. All rights reserved.
```

**Review by evargast — APPROVED** [review]
Love to see the AEP integration coming along 👍

**Review by bradgarropy — APPROVED** [review]

**Review by evargast — APPROVED** [review]

**Pull request merged**